openwisp_radius/tests/test_tasks.py

```diff
@@ -103,7 +103,9 @@ def test_delete_unverified_users(self):
         management.call_command('batch_add_users', **options)
         User.objects.update(date_joined=now() - timedelta(days=3))
         for user in User.objects.all():
-            RegisteredUser.objects.create(user=user, method='email', is_verified=False)
+            user.registered_user.is_verified = False
+            user.registered_user.method = 'email'
+            user.registered_user.save(update_fields=['is_verified', 'method'])
```

Comment:
the same thing has to be done in `openwisp_radius.tests.test_commands.TestCommands.test_unverified_users_command`